Pet-Clinic-Frontend/src/pages/Registration.tsx

```diff
@@ -1,49 +1,116 @@
-
 import { useState } from 'react';
-
-import './Registration.css'
+import './Registration.css';
 
 const Registration = () => {
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [error, setError] = useState<string | null>(null); //generic type <string | null> specify that the state managed by useState can be a string or null. Initial State (null)
+    const [submitted, setSubmitted] = useState(false);
+
+    const handleEmail = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setEmail(e.target.value);
+        setSubmitted(false);
+    }
+
+    const handlePassword = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setPassword(e.target.value);
```

Comment:
maybe use event instead of e as we are using in login page

Reply:
Hey @Normelik you can even use the suggest feature to make it easier for the PR authors. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request